health_check/views.py

```diff
@@ -29,8 +29,9 @@ def _run(plugin):
                 connection.close()
 
         with ThreadPoolExecutor(max_workers=len(plugins) or 1) as executor:
-            for ers in executor.map(_run, plugins):
-                errors.extend(ers)
+            for plugin, ers in zip(plugins, executor.map(_run, plugins)):
+                if plugin.critical:
```

Comment:
This is not what I was going for. Now you just ignore errors if they are not critical. So they will be not shown.